nova/virt/vmwareapi/vm_util.py

```diff
@@ -1593,7 +1593,7 @@ def _get_host_reservations_map(groups=None):
 # Only for satisfying the tests, and to ensure it is producing the same results
 def get_stats_from_cluster(session, cluster):
     return aggregate_stats_from_cluster(
-        get_stats_from_cluster_per_host(session, cluster))
+        get_hosts_with_stats(session, cluster))
 
 
 def aggregate_stats_from_cluster(host_stats):
```

Comment:
while you're at it, you could rename this parameter:

```suggestion
def aggregate_stats_from_cluster(hosts_with_stats):
```

(would also need two more renames below as well)